<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shp2Postgresql</class>
 <widget class="QMainWindow" name="Shp2Postgresql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shp2Postgresql</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>431</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="shapefilePathLabel">
         <property name="text">
          <string>Shapefile Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="shapefilePathLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="shapefilePathBrowseButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="tableNameLabel">
         <property name="text">
          <string>Tabel Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tableNameLineEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
